templates/rack_mini_profiler.rb

```diff
@@ -0,0 +1,5 @@
+if ENV["RACK_MINI_PROFILER"].to_i > 0
```

Comment:
whats the scenario where you wouldnt want this on? Are we concerned with the visual pollution? We could start it hidden (option-p, IIRC, toggles it).